packages/core/src/components/Label/Label.tsx

```diff
@@ -92,19 +95,27 @@ const Label: VibeComponent<LabelProps> & {
     );
 
     return (
-      <span
-        {...(isClickable && clickableProps)}
-        className={cx({ [styles.clickable]: isClickable }, overrideClassName)}
-        data-testid={dataTestId || getTestId(ComponentDefaultTestId.LABEL, id)}
-        ref={mergedRef}
-      >
-        <Text element="span" type={Text.types.TEXT2} className={classNames} color={Text.colors.ON_INVERTED}>
-          <Text element="span" type={Text.types.TEXT2} color={Text.colors.INHERIT}>
-            {text}
+      <LabelCelebrationAnimation active={celebration && kind === "line"}>
+        <span
+          {...(isClickable && clickableProps)}
+          className={cx({ [styles.clickable]: isClickable }, overrideClassName)}
+          data-testid={dataTestId || getTestId(ComponentDefaultTestId.LABEL, id)}
+          ref={mergedRef}
+        >
+          <Text
+            element="span"
+            type={Text.types.TEXT2}
+            className={classNames}
+            color={Text.colors.ON_INVERTED}
+            data-celebration-text={celebration}
```

Comment:
Need this to edit text, couldn't find a better way

Reply:
Why not class?

Reply:
I can't use a moduled class because I'm targeting the selector from the animation wrapper component